tests/unit/grading/test_jplag.py

```diff
@@ -0,0 +1,66 @@
+from tempfile import TemporaryDirectory
+
+from django.core.files.uploadedfile import SimpleUploadedFile
+from django.test import TestCase
+
+from inloop.grading.copypasta import jplag_check
+from inloop.solutions.models import SolutionFile
+
+from tests.unit.grading.mixins import PlagiatedSolutionsData
+
+FIBONACCI = """
```

Comment:
As discussed, we should add more sources to the test. I am thinking of two semantically different solutions and two solutions, one of which having extensive amounts of whitespaces. The whitespacing should not affect the plagiarism tests. The semantically different solutions should throw a similarity value less than, let's say 80%.

Reply:
Ok, good idea, but I would prefer creating another pull request for more elaborate tests and getting this PR finished quicker.